src/modm/platform/clock/sam_pmc/module.lb

```diff
@@ -15,16 +15,20 @@ def init(module):
     module.description = "Clock Generator (CKGR)"
 
 def prepare(module, options):
-    if not options[":target"].has_driver("pmc:samg*"):
+    if not (options[":target"].has_driver("pmc:samg*")
+        or options[":target"].has_driver("pmc:sam")):
```

Comment:
> Should the modm-devices PR be updated so that this can be "pmc:samv*",

Yes, please do so.

Reply:
Would you mind giving me a clue how such update would look like? :) I have no idea what do you guys want to change.

Reply:
I think this line needs to be changed to `"sam" + id.family` so that by default all driver types are namespaced properly until we can determine compatibility systematically.
https://github.com/modm-io/modm-devices/blob/develop/tools/generator/dfg/sam/sam_device_tree.py#L171

Reply:
Done.